### PR TITLE
Move Changelog to Module

### PR DESCRIPTION
**Pull request by unclebanks**

### CODE REVIEW

**Review by RedSparr0w — APPROVED** [review]
👍 Looks good to me, thanks for the PR!

**Comment by RedSparr0w** [comment]
Just checked a new game, it seems to break the intro.js stuff.

**Comment by unclebanks** [comment]
Crap, I figured that one might've. Is there a way I can fix it or should I just revert it for now?

**Comment by unclebanks** [comment]
removed the changes in regards to the information.ts. hopefully everything is fine now.

File: src/modules/changelog/ChangelogType.ts [hunk 1]
```diff
@@ -48,7 +43,7 @@ class ChangelogUpdate extends Changelog {
  * MINOR - Will increment for each feature refactor or large changes to a feature
  * PATCH - Increment for small changes, bugfixes, UI changes.
  */
-const changelogItems = [
+export const changelogItems = [
```
Comment (RedSparr0w) [inline src/modules/changelog/ChangelogType.ts:46]: Could we maybe move these to their own file?

Just seems a little odd to be importing them from the `ChangelogType` file.

Comment (unclebanks) [reply src/modules/changelog/ChangelogType.ts:46]: When i did that i was getting the complaints of circular dependencies. I can try to find a different way to do so. maybe i did it wrong in the first place?

Comment (unclebanks) [reply src/modules/changelog/ChangelogType.ts:46]: I tried changing it to have its own file or putting it in changelog but i just got complaints of not finding Change in Changelogtype and stuff. I can't figure out a better way to do this. I can close the PR if you'd prefer as I can't really change (or fix?) it.

**Pull request merged**